scss/login.scss

```diff
@@ -0,0 +1,225 @@
+body.modal-open {
+  #main-outlet {
+    min-height: 100vh;
+  }
+}
+
+#discourse-modal .create-account .modal-body {
+  max-height: none !important;
```

Comment:
When I do this, I found useful explicit say why you need the `!important`.
In that sense, people in the future is less prone to deal with this code as a black box that nobody can edit.

Reply:
This is to override Discourse code (they are using `!important` too). I'll put a reference to the code